vite.config.ts

```diff
@@ -1,14 +1,18 @@
-import path from "path"
-import { defineConfig } from 'vite'
-import tailwindcss from '@tailwindcss/vite'
-import react from '@vitejs/plugin-react'
+import path from 'path';
+import { defineConfig } from 'vite';
+import tailwindcss from '@tailwindcss/vite';
+import react from '@vitejs/plugin-react';
 
 // https://vite.dev/config/
 export default defineConfig({
-  plugins: [react(), tailwindcss()],
-  resolve: {
-    alias: {
-      "@": path.resolve(__dirname, "./src"),
-    },
-  },
-})
+	plugins: [react(), tailwindcss()],
+	resolve: {
+		alias: {
+			'@': path.resolve(__dirname, './src'),
```

Comment:
I'm not sure how this is working today. This repo has `"type": "module"` defined in the `package.json`, it should treat all .js/.ts files as ESM in which case `__dirname` does not exist: https://nodejs.org/docs/latest/api/esm.html#no-__filename-or-__dirname.

The docs suggest using `import.meta.dirname`, but that wasn't generally available until Node.js 24, so you will need to do the `const __dirname = dirname(fileURLToPath(import.meta.url));` hack.

What am I missing?

Reply:
https://v2.vitejs.dev/config/#config-file-resolving